src/agent/structures.ts

```diff
@@ -67,7 +67,7 @@ export interface GetChatResponse {
 }
 
 export interface ListArchivesParameters {
-  filters?: object;
+  filters?: ArchivesFilters;
```

Comment:
`ArchiveFilters` has required fields e.g. `agents`, `tags` etc. Before when we have `object` there developers could create any kind of object and type it `ArchivesFilters` manually:

```typescript
const p: ListArchivesParameters = { filters: {} as Partial<ArchiveFilters> };
```

Currently `ListArchivesParameters` needs to get `ArchivesFilters` (of course optionally, but still) and `ArchivesFilters` has many non-optional fields which really are optional in agent-api. I think you could type

```typescript
export interface ListArchivesParameters {
  filters?: Partial<ArchiveFilters>;
}
```

or set these fields optional.